pkg/sql/logictest/testdata/logic_test/txn

```diff
@@ -222,7 +222,7 @@ statement ok
 UPDATE kv SET v = 'b' WHERE k in ('a')
 
 statement error cannot change the isolation level of a running transaction
-SET TRANSACTION ISOLATION LEVEL SNAPSHOT
+SET transaction_isolation = 'snapshot'
```

Comment:
This test case duplicated the one before, so I'm just modifying it to suit my purposes instead of deleting it.